src/components/Single-Article.jsx

```diff
@@ -2,18 +2,20 @@ import { useEffect } from "react"
 import { useState } from "react"
 import { useParams } from "react-router-dom"
 import { getSingleArticle, getCommentsForArticle } from "../../api.get"
+import { patchVotes } from "../../api.patch"
 
 export const SingleArticle = () => {
 
-    const [article, setArticle] = useState(null)
-    const [comments, setComments] = useState([])
+    const [article, setArticle] = useState(null);
```

Comment:
remember to have a loading state for your articles as well as one for your comments too